test/connect-log4j.properties

```diff
@@ -41,3 +41,6 @@ log4j.appender.connectAppender.layout.ConversionPattern=${connect.log.pattern}
 
 log4j.logger.org.apache.zookeeper=ERROR
 log4j.logger.org.reflections=ERROR
+
+# enable debug level logging only for SF's KC
+log4j.logger.com.snowflake.kafka.connector=DEBUG
```

Comment:
Should this be in the release notes? Customers will see more and detailed logs, IMO it is worth mentioning

Reply:
Yes, I plan to add this in a miscellaneous debugging tips.

Reply:
https://snowflakecomputing.atlassian.net/browse/SNOW-368659